### PR TITLE
Add links to glossary

### PR DESCRIPTION
**Pull request by paulidale**

### CODE REVIEW

File: policies/design-process.md [hunk 1]
```diff
@@ -156,3 +156,5 @@ offered an invite to join the solution selection meeting.
 
 * The refined documented design should be available for public comment as a pull
 request for at least one week prior to eventual agreement of the solution
+
+[OtC]: https://github.com/openssl/general-policies/blob/master/policies/glossary.md#otc
```
Comment (t8m) [inline policies/design-process.md:160]: typo OTC

File: policies/api-compat.md [hunk 2]
```diff
@@ -14,11 +14,15 @@ includes, but is not limited to:
 - changing a macro to a function and
 - corrections of spelling.
 
-Only API additions are allowed in minor releases.
+Only API additions are allowed in [minor] releases.
```
Comment (mspncp) [inline policies/api-compat.md:17]:
```suggestion
Only API additions are allowed in [minor releases].
```

File: policies/api-compat.md [hunk 3]
```diff
@@ -14,11 +14,15 @@ includes, but is not limited to:
...
+Although the changes as listed above might be regarded as [ABI] compatible, they
 cause various possible breakage when building applications depending on these
 APIs and thus they are not allowed in minor releases.
 
 If necessary, a new API call can be added to implement the required changes in
 minor releases.
+
+[ABI]: https://github.com/openssl/general-policies/blob/master/policies/glossary.md#abi
+[API]: https://github.com/openssl/general-policies/blob/master/policies/glossary.md#api
+[minor]: https://github.com/openssl/general-policies/blob/master/policies/glossary.md#minor-release
```
Comment (mspncp) [inline policies/api-compat.md:28]:
```suggestion
[minor releases]: https://github.com/openssl/general-policies/blob/master/policies/glossary.md#minor-release
```

**Review by mspncp — APPROVED** [review]
Just a nit, at your discretion.

File: policies/api-compat.md (hunk 2 above)
Comment (paulidale) [reply policies/api-compat.md:17]: This cascades into too many other policies and lots of voting :)

Comment (mspncp) [reply policies/api-compat.md:17]: I don't get the point. I am just suggesting to change the link _text_ in a single file. How could this cascade?

Comment (paulidale) [reply policies/api-compat.md:17]: The target is linked the same way in other files.  We should change the link text for all other instances for consistency.

File: policies/testing.md [hunk 4]
```diff
@@ -71,3 +71,10 @@ check that there are sufficient tests and that all corner cases are covered.
 
 Where performance fixes are implemented consideration should be given to whether
 we need to add a performance test to the performance testing suite.
+
+
+[CHANGES]: https://github.com/openssl/general-policies/blob/master/policies/definitions.md#news
```
Comment (romen) [inline policies/testing.md:76]: Swap changes and news in the links

File: policies/documentation-policy.md [hunk 5]
```diff
@@ -120,3 +120,6 @@ However, common sense applies.  Always err in favour of including more
 comments than less or none.  Code that you've just written that is
 _obvious_ will not necessarily be to someone else two years later.
 
+
+[CHANGES]: https://github.com/openssl/general-policies/blob/master/policies/definitions.md#news
```
Comment (romen) [inline policies/documentation-policy.md:124]: News/changes are inverted in the links

**Review by romen — CHANGES_REQUESTED** [review]

File: policies/api-compat.md (hunk 2 above)
Comment (t8m) [reply policies/api-compat.md:17]: > lots of voting

There is no need to have a separate vote for every policy change. A single change that is voted on can affect multiple policy files.

**Comment by paulidale** [comment]
Vote for accepting the policy at the commit id 8385661d131113732f6feb714d4e78a5f7728489 has started.

**Comment by paulidale** [comment]
vote: [+1]

**Comment by mspncp** [comment]
vote: [+1]

**Comment by t-j-h** [comment]
Vote: [+1]

**Comment by t8m** [comment]
Vote: [+1]

**Comment by mattcaswell** [comment]
Vote: [+1]

**Comment by beldmit** [comment]
Vote: [+1]

**Comment by levitte** [comment]
Vote: [+1]

(with reservation; we may want to think a bit more about the strict language policy...  but I think that can be left for another PR and vote)

**Comment by romen** [comment]
Vote: [+0]

Note that as of 8385661d131113732f6feb714d4e78a5f7728489 (the commit we are voting on), the following links are broken:
- https://github.com/openssl/general-policies/blob/master/policies/definitions.md#changes
- https://github.com/openssl/general-policies/blob/master/policies/definitions.md#news
- https://github.com/openssl/general-policies/blob/master/policies/glossary.md#ldp (file exists, missing entry should be added via https://github.com/openssl/general-policies/pull/10)

**Comment by paulidale** [comment]
Those links should come good with both pull requests merged.
The vote here has passed, the vote on openssl/general-policies#10 is one shy of passing.

File: policies/api-compat.md (hunk 2 above)
Comment (slontis) [reply policies/api-compat.md:17]: This change seems weird - the glossary item is "minor releases" not "minor"

Comment (slontis) [reply policies/api-compat.md:17]: In the same way that you have "stable releases" below.

**Comment by slontis** [comment]
Vote: [+1]

**Comment by paulidale** [comment]
Vote results:
```
Topic: Accept the glossary links as at commit id 8385661
comment: this requires openssl/general-policies#10
Proposed by: Pauli
Issue link: https://github.com/openssl/technical-policies/pull/27
Public: yes
Opened: 2022-03-04
Closed: 2022-03-09
Accepted:  yes  (for: 8, against: 0, abstained: 1, not voted: 1)

  Dmitry     [+1]
  Matt       [+1]
  Pauli      [+1]
  Tim        [+1]
  Richard    [+1]
  Shane      [+1]
  Tomas      [+1]
  Kurt       [  ]
  Matthias   [+1]
  Nicola     [+0]
```

**Comment by kroeckx** [comment]
vote +1
